app/importers/unpaywall_publication_importer.rb

```diff
@@ -89,7 +89,14 @@ def update_publication(publication, unpaywall_json)
     end
 
     def query_unpaywall_for(publication)
-      find_url = URI::DEFAULT_PARSER.escape("https://api.unpaywall.org/v2/#{publication.doi_url_path}?email=openaccess@psu.edu")
+      unpaywall_url = "https://api.unpaywall.org/v2/#{publication.doi_url_path}?email=openaccess@psu.edu"
+
+      find_url = if %w[[ ]].any? { |s| publication.doi_url_path.include?(s) }
+                   Addressable::URI.encode(unpaywall_url)
+                 else
+                   URI::DEFAULT_PARSER.escape(unpaywall_url)
```

Comment:
@ajkiessl we can change this to Addressable::URI.encode, it works it that way but I wanted to get your thoughts, right now I just did it for the URL that has brackets in them.

Reply:
I think it's fine to just use the Addressable::URI.encode.  If you run the Unpaywall import with just the Addressable code, does it cause any errors?

Reply:
Also, idk if this would help with anything, but we might not need to escape the entire url.  We might just need to escape the `#{publication.doi_url_path}` part.